scripts/install.js

```diff
@@ -37,7 +37,7 @@ var ninja_vendor_dir = path.join(root_dir, 'vendor', 'ninja-build')
 
 function build_ninja() {
     console.log('No prebuilt Ninja, building Ninja now')
-    var build_ninja_command = "tar -xf  ninja-1.7.2.tar.gz  && cd  ninja-1.7.2  && ./configure.py --bootstrap "
+    var build_ninja_command = "tar -xzf  ninja-1.7.2.tar.gz  && cd  ninja-1.7.2  && ./configure.py --bootstrap "
```

Comment:
I don't think the -z flag works on osx with that bsdtar (https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/tar.1.html says it's in c mode only.) will double check when I get to work tomorrow though.